backend/functions/lib/controllers/case-history/case-history.controller.ts

```diff
@@ -20,22 +16,21 @@ export class CaseHistoryController {
 
   public async getCaseHistory(
     context: ApplicationContext,
-    request: GetCaseHistoryRequest,
-  ): Promise<ResponseBody<CaseHistory[]>> {
+    request: CamsHttpRequest,
+  ): Promise<CamsHttpResponseInit<CaseHistory[]>> {
     try {
-      const caseHistory = await this.useCase.getCaseHistory(context, request.caseId);
-      return {
-        meta: {
-          isPaginated: false,
-          self: context.req.url,
+      const caseHistory = await this.useCase.getCaseHistory(context, request.params.id);
```

Comment:
same here.  The request is embedded in the context, so we don't need to pass the id to getCaseHistory.